File: src/main/resources/static/js/UserService.js
angular.module('employee.services', []).factory('UserService',
		[ "$http", function($http) {
            var service = {};
            //getuser by id

            service.getUserById = function(userId) {
				
				return $http.get("/user/getUser/1");
			}

            // //get all users
			 service.getAllUsers = function() {
                 return $http.get("/user/getAllUsers");
                 //defining 
			}
			return service;
		} ]);

File: src/main/java/com/example/employee/service/UserService.java
package com.example.employee.service;

import java.util.List;

import com.example.employee.entity.User;

public interface UserService {

    //users take from entities and entities take from dto with the help of converters and send to the controllers
    //list of users from entities 
    List<User> getAllUsers();

   //get user by id from entities
   User getUserById(Integer userId);
}